rlkit/core/rl_algorithm.py

```diff
@@ -225,6 +224,7 @@ def pretrain(self):
         """
         pass
 
+    # do we care about the flexibility to pass in an agent?
```

Comment:
no I don't think we do since we only have one policy at any given time

Reply:
removed agent arg 
0d9d63d